From ab830ecbae596d985f4fc02198a114bb46514635 Mon Sep 17 00:00:00 2001
From: Quoc Ngo <quoc.ngo.wz@rvc.renesas.com>
Date: Fri, 29 Sep 2017 16:11:24 +0700
Subject: [PATCH] qt-hmi-demo: Fix missing avatar picture

Fix missing avatar picture on cinematicexperience app dislay on LVDS screen

Signed-off-by: Quoc Ngo <quoc.ngo.wz@rvc.renesas.com>
---
 Qt5_CinematicExperience.qml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Qt5_CinematicExperience.qml b/Qt5_CinematicExperience.qml
index 58b8b6d..0bcd8f4 100644
--- a/Qt5_CinematicExperience.qml
+++ b/Qt5_CinematicExperience.qml
@@ -1,11 +1,12 @@
 import QtQuick 2.0
 import "content"
+import QtQuick.Window 2.1
 
 Item {
     id: mainWindow
 
-    width: 1920
-    height: 1080
+    width: Screen.desktopAvailableWidth < 1920 ? 800 : 1920
+    height: Screen.desktopAvailableHeight < 1080 ? 480 : 1080
 
     QtObject {
         id: settings
-- 
1.9.1

